extras/travis_build.sh

```diff
@@ -46,21 +46,21 @@ else
 fi
 
 if [ "$xPYTHON" = "1" ] ; then
-    if [ "$TRAVIS_PYTHON_VERSION" = "3.6.10" ]; then
-	PYPREFIX="/opt/python/3.6.10"
-	CONFIGURE_PYTHON="\
-            -DPYTHON_LIBRARY=$PYPREFIX/lib/libpython3.6m.so
-            -DPYTHON_INCLUDE_DIR=$PYPREFIX/include/python3.6m/"
-    fi
-    if [[ $OSTYPE = darwin* ]]; then
-	PYPREFIX="/usr/local/Cellar/python@3.8/3.8.4/Frameworks/Python.framework/Versions/3.8"
-	CONFIGURE_PYTHON="\
-            -DPYTHON_LIBRARY=$PYPREFIX/lib/libpython3.8.dylib
-            -DPYTHON_INCLUDE_DIR=$PYPREFIX/include/python3.8"
-    fi
-
-    mkdir -p $HOME/.matplotlib
-    echo "backend : svg" > $HOME/.matplotlib/matplotlibrc
+   PYTHON_LIB_DIR=`python3 -c "import sysconfig; print(sysconfig.get_path('platstdlib'))"`
+   PYTHON_INCLUDE_DIR=`python3 -c "import sysconfig; print(sysconfig.get_path('include'))"` 
```

Comment:
Maybe it would be good (for debugging now and for diagnosis later on) if you printed these paths and also what is in them, i.e.
```suggestion
   PYTHON_LIB_DIR=`python3 -c "import sysconfig; print(sysconfig.get_path('platstdlib'))"`
   echo "-> Detected PYTHON_LIB_DIR=$PYTHON_LIB_DIR"
   ls "$PYTHON_LIB_DIR"
   PYTHON_INCLUDE_DIR=`python3 -c "import sysconfig; print(sysconfig.get_path('include'))"`
   echo "-> Detected PYTHON_INCLUDE_DIR=$PYTHON_INCLUDE_DIR"
   ls "$PYTHON_INCLUDE_DIR"
```